types/msgpack5/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "private": true,
     "dependencies": {
-        "safe-buffer": "*"
+        "bl": ">=5.1.0"
```

Comment:
This should be `~5.1.2` because `5.1.2` is the first version of `bl` that contains its own typings.